contrib/epee/include/net/abstract_tcp_server2.inl

```diff
@@ -103,6 +102,18 @@ namespace net_utils
 		m_local(false),
 		m_ready_to_close(false)
   {
+    constexpr size_t N = std::mt19937_64::state_size;
+
+    std::random_device dev;
+    std::seed_seq::result_type rand[N]{};  // Use complete bit space
+
+    std::generate_n(rand, N, std::ref(dev));
+    std::seed_seq seed(rand, rand + N);
+    m_rng.seed(seed);
+
+    // Diffuse the initial state in case it has low quality
+    m_rng.discard(1000);
```

Comment:
Surely if the initial state has low quality, the rest will not have higher quality ?

Reply:
If the initial state has all 0s and only a few numbers which are not 0, it will take up to 10,000 steps to recover from it: https://stackoverflow.com/questions/44841819/mersenne-twister-limitations-used-in-agent-based-models
I limit it to 1000 here as a bare minimum and considering that std::random_device usually doesn't give out all 0s.

Reply:
So for the purposes of this, you define quality as "passes randomness tests", rather than "has enough entropy" ?

Reply:
Nevermind, it's just for sleeping in network code so we don't really care anyway.